lib/helpbase.gi

```diff
@@ -498,6 +498,10 @@ InstallGlobalFunction(HELP_ADD_BOOK, function( short, long, dir )
   sortfun := function(a, b)
     local main, pa, pb;
     main := ["tutorial", "reference", "changes"];
+    if ForAny( GAPInfo.RootPaths,
+               rp -> IsExistingFile( Concatenation( rp, "/doc/dev" ) ) ) then
+      Add( main, "development" );
+    fi;
```

Comment:
This re-creates `main` each time `sortfun` is called; when there are many packages, that can be a few hundred times. Why not move `main` and its initialization out of `sortfun`, directly into `HELP_ADD_BOOK`? It would still be visible inside `sortfun`.

Of course this really goes beyond this PR, but as they say "Always leave the campground cleaner than you found it" ;-).